controllers/endpoint.controller.js

```diff
@@ -115,4 +115,26 @@ exports.patchArticlesByID = (req, res, next) => {
     .catch((err) => {
         next(err)
     });
+};
+
+exports.removeCommentByID = (req, res, next) => {
+
+    let chosenComment = req.params.comment_id
+
+    deleteCommentByID(chosenComment)
+    .then(( rowCount ) => {
+
+        if (rowCount === 0) {
+            return Promise.reject( { status: 404, msg: `Comment ${chosenComment} did not exist or was already deleted!`} )
+        }
+        else if (rowCount === 1) {
+            res.status(204).send() //nothing to send only the status
+        }
+        else { 
+            return Promise.reject( { status: 500, msg: "Unexpected error!" } )
```

Comment:
You shouldn't need to put a 500 handler in here, it should be at the bottom of your error handling middleware.